### PR TITLE
log-backup: allow observer hibernate when there isn't any task

### LINKED ISSUE #14012: Logging from backup-stream is noisy
### What version of TiKV are you using?

```
TiKV 
Release Version:   6.5.0
Edition:           Community
Git Commit Hash:   47b81680f75adc4b7200480cea5dbe46ae07c4b5
Git Commit Branch: heads/refs/tags/v6.5.0
UTC Build Time:    2022-12-21 09:03:22
Rust Version:      rustc 1.67.0-nightly (96ddd32c4 2022-11-14)
Enable Features:   pprof-fp jemalloc mem-profiling portable sse test-engine-kv-rocksdb test-engine-raft-raft-engine cloud-aws cloud-gcp cloud-azure
Profile:           dist_release
```

### What operating system and CPU are you using?

Fedora Linux 37, Linux 6.0.12, Intel Core i7

### Steps to reproduce
Run a TiUP Playground. Then run `tiup bench tpcc prepare`.

### What did you expect?
No noisy logging

### What did happened?

```shell
grep 'subscription_track.rs:159' .tiup/data/TRUCY5d/tikv-0/tikv.log
```

```json
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:02.301 +01:00","new_region":"id: 2 start_key: 748000FFFFFFFFFFFFFE00000000000000F8 region_epoch { conf_ver: 1 version: 2 } peers { id: 3 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:02.311 +01:00","new_region":"id: 4 start_key: 748000FFFFFFFFFFFFFD00000000000000F8 end_key: 748000FFFFFFFFFFFFFE00000000000000F8 region_epoch { conf_ver: 1 version: 3 } peers { id: 5 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:02.322 +01:00","new_region":"id: 6 start_key: 748000FFFFFFFFFFFFFC00000000000000F8 end_key: 748000FFFFFFFFFFFFFD00000000000000F8 region_epoch { conf_ver: 1 version: 4 } peers { id: 7 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:02.344 +01:00","new_region":"id: 8 start_key: 748000FFFFFFFFFFFFFB00000000000000F8 end_key: 748000FFFFFFFFFFFFFC00000000000000F8 region_epoch { conf_ver: 1 version: 5 } peers { id: 9 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:03.766 +01:00","new_region":"id: 10 start_key: 7480000000000000FF0400000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 6 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:03.912 +01:00","new_region":"id: 10 start_key: 7480000000000000FF0600000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 7 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:04.064 +01:00","new_region":"id: 10 start_key: 7480000000000000FF0800000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 8 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:04.345 +01:00","new_region":"id: 10 start_key: 7480000000000000FF0A00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 9 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:04.501 +01:00","new_region":"id: 10 start_key: 7480000000000000FF0C00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 10 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:04.646 +01:00","new_region":"id: 10 start_key: 7480000000000000FF0E00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 11 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:04.818 +01:00","new_region":"id: 10 start_key: 7480000000000000FF1000000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 12 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:05.021 +01:00","new_region":"id: 10 start_key: 7480000000000000FF1200000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 13 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:05.170 +01:00","new_region":"id: 10 start_key: 7480000000000000FF1400000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 14 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:05.315 +01:00","new_region":"id: 10 start_key: 7480000000000000FF1600000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 15 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:05.462 +01:00","new_region":"id: 10 start_key: 7480000000000000FF1800000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 16 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:05.611 +01:00","new_region":"id: 10 start_key: 7480000000000000FF1A00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 17 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:05.753 +01:00","new_region":"id: 10 start_key: 7480000000000000FF1C00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 18 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:05.896 +01:00","new_region":"id: 10 start_key: 7480000000000000FF1E00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 19 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:06.066 +01:00","new_region":"id: 10 start_key: 7480000000000000FF2000000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 20 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:06.210 +01:00","new_region":"id: 10 start_key: 7480000000000000FF2200000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 21 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:06.351 +01:00","new_region":"id: 10 start_key: 7480000000000000FF2400000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 22 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:06.504 +01:00","new_region":"id: 10 start_key: 7480000000000000FF2600000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 23 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:06.661 +01:00","new_region":"id: 10 start_key: 7480000000000000FF2800000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 24 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:06.812 +01:00","new_region":"id: 10 start_key: 7480000000000000FF2A00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 25 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:06.956 +01:00","new_region":"id: 10 start_key: 7480000000000000FF2C00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 26 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:07.114 +01:00","new_region":"id: 10 start_key: 7480000000000000FF2E00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 27 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:07.269 +01:00","new_region":"id: 10 start_key: 7480000000000000FF3000000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 28 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:07.411 +01:00","new_region":"id: 10 start_key: 7480000000000000FF3200000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 29 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:07.565 +01:00","new_region":"id: 10 start_key: 7480000000000000FF3400000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 30 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:07.731 +01:00","new_region":"id: 10 start_key: 7480000000000000FF3600000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 31 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:07.890 +01:00","new_region":"id: 10 start_key: 7480000000000000FF3800000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 32 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:08.044 +01:00","new_region":"id: 10 start_key: 7480000000000000FF3A00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 33 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:08.192 +01:00","new_region":"id: 10 start_key: 7480000000000000FF3C00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 34 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:08.341 +01:00","new_region":"id: 10 start_key: 7480000000000000FF3E00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 35 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:08.482 +01:00","new_region":"id: 10 start_key: 7480000000000000FF4000000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 36 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:08.629 +01:00","new_region":"id: 10 start_key: 7480000000000000FF4200000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 37 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:08.806 +01:00","new_region":"id: 10 start_key: 7480000000000000FF4400000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 38 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:08.959 +01:00","new_region":"id: 10 start_key: 7480000000000000FF4600000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 39 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:09.105 +01:00","new_region":"id: 10 start_key: 7480000000000000FF4800000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 40 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:09.256 +01:00","new_region":"id: 10 start_key: 7480000000000000FF4A00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 41 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:09.410 +01:00","new_region":"id: 10 start_key: 7480000000000000FF4C00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 42 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:25:09.569 +01:00","new_region":"id: 10 start_key: 7480000000000000FF4E00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 43 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:31:58.991 +01:00","new_region":"id: 10 start_key: 7480000000000000FF5000000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 44 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:31:59.173 +01:00","new_region":"id: 10 start_key: 7480000000000000FF5200000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 45 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:31:59.370 +01:00","new_region":"id: 10 start_key: 7480000000000000FF5400000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 46 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:31:59.528 +01:00","new_region":"id: 10 start_key: 7480000000000000FF5600000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 47 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:31:59.704 +01:00","new_region":"id: 10 start_key: 7480000000000000FF5800000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 48 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:31:59.858 +01:00","new_region":"id: 10 start_key: 7480000000000000FF5A00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 49 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:32:00.012 +01:00","new_region":"id: 10 start_key: 7480000000000000FF5C00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 50 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:32:00.176 +01:00","new_region":"id: 10 start_key: 7480000000000000FF5E00000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 51 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:32:00.376 +01:00","new_region":"id: 10 start_key: 7480000000000000FF6000000000000000F8 end_key: 748000FFFFFFFFFFFFFB00000000000000F8 region_epoch { conf_ver: 1 version: 52 } peers { id: 11 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:32:30.596 +01:00","new_region":"id: 104 start_key: 7480000000000000FF5E5F720380000000FF0000000303800000FF0000007EE5000000FC end_key: 7480000000000000FF6000000000000000F8 region_epoch { conf_ver: 1 version: 53 } peers { id: 105 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:32:50.635 +01:00","new_region":"id: 104 start_key: 7480000000000000FF5E5F720380000000FF0000000503800000FF000000FDC6000000FC end_key: 7480000000000000FF6000000000000000F8 region_epoch { conf_ver: 1 version: 54 } peers { id: 105 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:33:20.642 +01:00","new_region":"id: 104 start_key: 7480000000000000FF5E5F720380000000FF0000000703800000FF0000017CAC000000FC end_key: 7480000000000000FF6000000000000000F8 region_epoch { conf_ver: 1 version: 55 } peers { id: 105 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:34:48.657 +01:00","new_region":"id: 102 start_key: 7480000000000000FF5C5F720380000000FF0000000303800000FF0000000005038000FF0000000002600380FF0000000000000100FE end_key: 7480000000000000FF5E00000000000000F8 region_epoch { conf_ver: 1 version: 52 } peers { id: 103 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:35:48.129 +01:00","new_region":"id: 94 start_key: 7480000000000000FF545F720380000000FF0000000403800000FF0000000004038000FF0000000004050000FD end_key: 7480000000000000FF5600000000000000F8 region_epoch { conf_ver: 1 version: 48 } peers { id: 95 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:35:48.701 +01:00","new_region":"id: 102 start_key: 7480000000000000FF5C5F720380000000FF0000000503800000FF0000000009038000FF0000000005D60380FF0000000000000200FE end_key: 7480000000000000FF5E00000000000000F8 region_epoch { conf_ver: 1 version: 53 } peers { id: 103 store_id: 1 }"}
{"level":"WARN","caller":"subscription_track.rs:159","message":"backup stream observer refreshing void subscription.","time":"2022/12/30 09:36:58.724 +01:00","new_region":"id: 102 start_key: 7480000000000000FF5C5F720380000000FF0000000803800000FF0000000003038000FF0000000009570380FF0000000000000B00FE end_key: 7480000000000000FF5E00000000000000F8 region_epoch { conf_ver: 1 version: 54 } peers { id: 103 store_id: 1 }"}
```

This is with log format set to `json`.

For users of TiKV it isn't clear if this indicates a problem or not. Note that these are warning messages.

If these message are normal and expected, then the level should probably be `INFO`.

Note that this is a TiUP playground without any backup or PITR going on.

### PR DESCRIPTION
**Pull request by YuJuncen**
Signed-off-by: hillium <yujuncen@pingcap.com>

### What is changed and how it works?

Issue Number: Close #14012

What's Changed:
This PR added a "hibernate mode" for the log backup observer: while there isn't any task, it won't emit leader drop or region change events. So some verbose logs can be omitted.

```commit-message
Added a "hibernate mode" for the log backup observer: while there isn't any task, it won't emit leader drop or region change events. So some verbose logs can be omitted.
```

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]

File: components/sst_importer/src/sst_importer.rs [hunk 1]
```diff
@@ -438,17 +438,28 @@ impl SstImporter {
 
         if self.import_support_download() {
             let shrink_file_count = shrink_files.len();
-            info!("shrink space by tick"; "shrink files count" => shrink_file_count, "retain files count" => retain_file_count);
-
+            debug!("shrink space by tick"; "shrink files count" => shrink_file_count, "retain files count" => retain_file_count);
```
Comment (joccau) [inline components/sst_importer/src/sst_importer.rs:441]: Maybe the tick() is called every 10 seconds, the log is not mush.

File: components/backup-stream/src/observer.rs [hunk 2]
```diff
@@ -149,7 +156,7 @@ impl<E: KvEngine> CmdObserver<E> for BackupStreamObserver {
 
 impl RoleObserver for BackupStreamObserver {
     fn on_role_change(&self, ctx: &mut ObserverContext<'_>, r: &RoleChange) {
-        if r.state != StateRole::Leader {
+        if r.state != StateRole::Leader && !self.is_hibernating() {
```
Comment (joccau) [inline components/backup-stream/src/observer.rs:159]: We need to test the full process containing the log-backup and PITR, check data consistency finally, after this PR is merged into master.

**Review by joccau — COMMENTED** [review]
Rest LGTM

**Review by Leavrth — APPROVED** [review]

**Review by joccau — APPROVED** [review]
LGTM

**Review by hicqu — APPROVED** [review]

**Pull request merged**